### ISSUE TITLE
damn yarik and his git-annex caused annexing of web UI assets etc

### ISSUE THREAD

**Issue by yarikoptic** [datalad-datasets/ohbm2020-posters#144]
those cannot work I believe with gh-pages now. 

```
$> git grep MD5E
.gitattributes:* annex.backend=MD5E
assets/images/favicon-16x16.png:/annex/objects/MD5E-s272--0dc060c335e46489212c626090266fb0.png
assets/images/favicon.ico:/annex/objects/MD5E-s15086--477c78bf3fd5d8eb7901272af1276697.ico
assets/images/sort_asc.png:/annex/objects/MD5E-s158--a8dcf333411ca3c52a00e878cbb42e86.png
assets/images/sort_both.png:/annex/objects/MD5E-s199--6c505aa30e5b18c4b561e7f49d0fc053.png
assets/images/sort_desc.png:/annex/objects/MD5E-s158--1fc418e33fd5a687290258b23fac4e98.png
static/favicon-16x16.png:/annex/objects/MD5E-s272--0dc060c335e46489212c626090266fb0.png
static/favicon.ico:/annex/objects/MD5E-s15086--477c78bf3fd5d8eb7901272af1276697.ico
ui/static/favicon-16x16.png:/annex/objects/MD5E-s272--0dc060c335e46489212c626090266fb0.png
ui/static/favicon.ico:/annex/objects/MD5E-s15086--477c78bf3fd5d8eb7901272af1276697.ico
```

Dear @soichih,  Please `git annex unannex` those files and commit them directly to git.  I just pushed e54a711a1994fcffbf49c81ac4ed904ad98bb4f8 so `git add ` and even `git annex add` should add them to git if you perform unannex after updating your gh-pages branch to that state.  Then also git add `.noannex` file and do `git annex uninit` -- I think this one should stay pure git, and I will add a submodule with PDFs under annex (I did add them directly here but then decided against that -- will redo in a separate repo)

**Comment by soichih** [comment]
I still see those files on gh-pages ?
> https://github.com/soichih/ohbm2020-posters/tree/gh-pages/assets/images
> https://github.com/datalad-datasets/ohbm2020-posters/tree/gh-pages/assets/images

I am probably not understanding the problem, but are you saying those files disappeared from gh-pages somehow??

**Comment by yarikoptic** [comment]
check the content of any of those files -- it will be a "git link" which `grep` output above shows.

**Comment by yarikoptic** [comment]
it is binary files committed into git annex in "unlocked" mode.  They will be "real files" (not symlinks, like in the regular "locked" mode) if you do `git annex get` on them.

**Comment by soichih** [comment]
Oh I see. Sorry I am slow. 

I am seeing this error message.

```
hayashis@haswell:~/git/ohbm2020-posters(gh-pages*) $ git annex get assets/images/favicon-16x16.png
get assets/images/favicon-16x16.png (not available) 
  No other repository is known to contain the file.

  (Note that these git remotes have annex-ignore set: origin upstream)
failed
git-annex: get: 1 failed

```

and on another machine

```
hayashis@dev1:~/git/ohbm2020-posters(gh-pages*) $ git annex get assets/images/favicon-16x16.png
git-annex: Repository version 8 is not supported. Upgrade git-annex.
```

I am not sure if I can help you there?

**Comment by yarikoptic** [comment]
oh -- twice my bad: I looked only at the last commit belonging to you so assumed that it was you who fell into the trap, but it was originally me!
```
$> git grep -l MD5E- | xargs git log --stat
commit 47cc5fdd3297ebdb65e60aba1da863368e43e31a
Author: Soichi Hayashi <hayashis@iu.edu>
Date:   Sun Jun 28 16:55:29 2020 -0400

    moved new vue ui structure to gh-pages.

 static/favicon-16x16.png    | 1 +
 static/favicon.ico          | 1 +
 ui/static/favicon-16x16.png | 1 +
 ui/static/favicon.ico       | 1 +
 4 files changed, 4 insertions(+)

commit bef67880dbb2d3a7ca21c0799e6a898c43a0ee23
Author: Yaroslav Halchenko <debian@onerussian.com>
Date:   Tue Jun 23 22:21:05 2020 -0400

    index.html and assets/ from datalad datasets

 assets/images/favicon-16x16.png | 1 +
 assets/images/favicon.ico       | 1 +
 assets/images/sort_asc.png      | 1 +
 assets/images/sort_both.png     | 1 +
 assets/images/sort_desc.png     | 1 +
 5 files changed, 5 insertions(+)
```
I will fix it up tomorrow ;)

**Comment by soichih** [comment]
On my dev server (where I am running the web socket backend..) I am seeing some error message from git-annex.

```
hayashis@dev1:~/git/ohbm2020-posters/backend(gh-pages*) $ git commit -av
git-annex: Repository version 8 is not supported. Upgrade git-annex.
error: external filter 'git-annex smudge --clean -- %f' failed 1
error: external filter 'git-annex smudge --clean -- %f' failed
git-annex: Repository version 8 is not supported. Upgrade git-annex.
error: external filter 'git-annex smudge --clean -- %f' failed 1
error: external filter 'git-annex smudge --clean -- %f' failed
git-annex: Repository version 8 is not supported. Upgrade git-annex.
error: external filter 'git-annex smudge --clean -- %f' failed 1
error: external filter 'git-annex smudge --clean -- %f' failed
git-annex: Repository version 8 is not supported. Upgrade git-annex.
error: external filter 'git-annex smudge --clean -- %f' failed 1
error: external filter 'git-annex smudge --clean -- %f' failed
git-annex: Repository version 8 is not supported. Upgrade git-annex.
```

I tried a few things to get gix-annex up-to-date, but so far I am not having much luck.

```
hayashis@dev1:~/git/ohbm2020-posters/backend(gh-pages*) 1 sudo apt-cache policy git-annex
git-annex:
  Installed: (none)
  Candidate: 6.20180227-1
  Version table:
     6.20180227-1 500
        500 http://zone-r2.clouds.archive.ubuntu.com/ubuntu bionic/universe amd64 Packages
        100 /var/lib/dpkg/status

hayashis@dev1:~/git/ohbm2020-posters/backend(gh-pages*) $ sudo apt-cache policy git-annex-standalone
git-annex-standalone:
  Installed: 7.20190819+git2-g908476a9b-1~ndall+1
  Candidate: 7.20190819+git2-g908476a9b-1~ndall+1
  Version table:
 *** 7.20190819+git2-g908476a9b-1~ndall+1 500
        500 http://neuro.debian.net/debian bionic/main amd64 Packages
        100 /var/lib/dpkg/status
You have new mail in /var/mail/hayashis

```

I have neurodebian installed, but it looks like git-annex is still coming from the ubuntu repo. Do I need to install it from the source?

**Comment by yarikoptic** [comment]
oh sorry -- forgot to fix this one up.  I will do that now. there should be no "annex" in this repo - not needed/desired

**Comment by yarikoptic** [comment]
pushed directly 2d366f2665c6ee3ac04a08d9f58b0d28ae8a5a36 to fix this and then also f87f6e2c1b8981ee0ec51f35f1cd6bbf679f9ff7 to add `.noannex` file.  Please 'git pull' into your clone and `git annex uninit`

**Comment by effigies** [comment]
Looks like this is resolved. Reopen if something left to do.
